tests/lib/KeyStoreIndexeddb.spec.js

```diff
@@ -0,0 +1,140 @@
+describe('KeyStore', () => {
+    it('is a singleton', () => {
+        const instance1 = KeyStore.instance;
+        const instance2 = KeyStore.instance;
+        expect(instance1).toBe(instance2);
+    });
+
+    it('connection can be opened and closed', async () => {
```

Comment:
Naming should follow the "it ..." syntax, for example **it** `can open and close a connection`.

Reply:
The test name combines to "KeyStore connection can be opened and closed".
But your suggestion also works